### PR TITLE
Added home page

### PR DESCRIPTION
**Pull request by SHIKHASINGH1506**
Modified home page of component library

### CODE REVIEW

File: docs-css/docs-nav.css [hunk 1]
```diff
@@ -1,96 +1,151 @@
...
+  width: 1.3rem;
+  height: 1.3rem;
+  font-size: 1.1rem;
+}
+.search-bar {
+  display: flex;
+  align-items: center;
+  flex-grow: 1;
+  position: relative;
+  background-color: #f7f7fd;
```
Comment (HarshnaNHaswani) [inline docs-css/docs-nav.css:81]: use variables for color

File: docs-css/docs-nav.css [hunk 2]
```diff
@@ -1,96 +1,151 @@
...
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  top: 0px;
+  left: 0;
+}
+.text-overlay-title {
+  color: black;
```
Comment (HarshnaNHaswani) [inline docs-css/docs-nav.css:119]: maintain same format for color code

File: docs-css/docs-nav.css [hunk 3]
```diff
@@ -1,96 +1,151 @@
...
+  .sidebar-toggler {
+    display: block;
+  }
 }
-@media only screen and (min-width: 601px) {
-  .navbar-collapse {
-    flex-basis: auto;
+/* media query */
+@media screen and (max-width: 426px) {
+  .text-overlay .text-overlay-title {
```
Comment (HarshnaNHaswani) [inline docs-css/docs-nav.css:145]: use ems for media queries

File: docs-css/docs-sidebar.css [hunk 4]
```diff
@@ -1,7 +1,7 @@
 .sidebar {
   grid-area: sidebar;
   position: sticky;
-  top: 0;
+  top: 62px;
```
Comment (HarshnaNHaswani) [inline docs-css/docs-sidebar.css]: use responsive units

File: docs.html [hunk 5]
```diff
@@ -30,48 +30,84 @@
...
+              class="sidebar-toggler"
+              id="sidebar-toggler"
+              viewBox="0 0 100 40"
+              width="40"
+              height="40"
+            >
+              <rect width="60" height="10"></rect>
+              <rect y="20" width="60" height="10"></rect>
+              <rect y="40" width="60" height="10"></rect>
+            </svg> -->
```
Comment (HarshnaNHaswani) [inline docs.html:46]: remove commented code

File: style.css [hunk 6]
```diff
@@ -5,7 +5,7 @@
 @import "./docs-css/docs-body.css";
 @import "./docs-css/docs-code.css";
 @import "./docs-css/docs-footer.css";
-@import "./utility.css";
+/* @import "./utility.css"; */
```
Comment (HarshnaNHaswani) [inline style.css:8]: remove comments

File: index.html [hunk 7]
```diff
@@ -20,48 +20,82 @@
...
+              class="sidebar-toggler"
+              id="sidebar-toggler"
+              viewBox="0 0 100 40"
+              width="40"
+              height="40"
+            >
+              <rect width="60" height="10"></rect>
+              <rect y="20" width="60" height="10"></rect>
+              <rect y="40" width="60" height="10"></rect>
+            </svg> -->
```
Comment (HarshnaNHaswani) [inline index.html:38]: remove comments

**Review by HarshnaNHaswani — COMMENTED** [review]
Few suggestions, rest good to go

File: docs-css/docs-nav.css [hunk 8]
```diff
@@ -1,96 +1,151 @@
-/* header */
-.navbar {
+.navbar-home {
+  padding: 0.7rem 0rem;
+  box-shadow: 0px 0px 10px 4px rgb(0 0 0 / 10%);
```
Comment (anupkrbid) [inline docs-css/docs-nav.css:3]: You can use variables for these complex values like box-shadow so you can reuse it and have less chance of mistakes.

File: docs-css/docs-nav.css (hunk 1 above)
Comment (anupkrbid) [reply docs-css/docs-nav.css:81]: agreed

**Comment by shlok30** [comment]
Add some margin between documentation button and text above it

File: docs.html (hunk 5 above)
Comment (anupkrbid) [inline docs.html:46]: when using SVG prefer storing it in a separate file in assets and using an IMG tag. unless you are manipulating the value in the svg with js

**Comment by shlok30** [comment]
Rest looks good to merge

File: docs.html [hunk 9]
```diff
@@ -30,48 +30,84 @@
...
+                  <i class="fab fa-linkedin-in" aria-hidden="true"></i>
+                </a>
               </li>
               <li class="nav-item">
-                <span class="nav-item-link">
-                  <i class="fab fa-twitter p-4" aria-hidden="true"></i>
-                </span>
+                <a
+                  class="nav-icon-link badge-container no-link-style"
+                  href="https://github.com/SHIKHASINGH1506"
```
Comment (anupkrbid) [inline docs.html:76]: add `target="_blank" or else the URL will open in the same tab the user is on, which is not a good UX in this case

**Review by anupkrbid — COMMENTED** [review]
over all good, just check the suggestions.

**Pull request merged**